src/topology/instances/ennreal.lean

```diff
@@ -1149,6 +1149,24 @@ lemma tsum_le_of_sum_range_le {f : ℕ → ℝ≥0∞} {c : ℝ≥0∞}
   (h : ∀ n, ∑ i in finset.range n, f i ≤ c) : ∑' n, f n ≤ c :=
 tsum_le_of_sum_range_le ennreal.summable h
 
+lemma has_sum_lt {f g : α → ℝ≥0∞} {sf sg : ℝ≥0∞} {i : α} (h : ∀ (a : α), f a ≤ g a)
+  (hi : f i < g i) (hsg : sg ≠ ⊤) (hf : has_sum f sf) (hg : has_sum g sg) : sf < sg :=
```

Comment:
it should be enough to assume the weaker assumption `sf ≠ ⊤`, here and in the next lemma.

Reply:
Changed this with a quick `by_cases` proof